api/v1alpha4/conversion.go

```diff
@@ -15,19 +15,3 @@ limitations under the License.
 */
 
 package v1alpha4
-
-import (
```

Comment:
There is nothing left in this file, could be removed in a follow up PR.

Reply:
Done